app/helpers/validation_search.py

```diff
@@ -203,6 +217,7 @@ def srid(self, value):
         if value in map(str, SUPPORTED_OUTPUT_SRS):
             self._srid = int(value)
         elif value is not None:
+            logger.warning("Unsupported spatial reference %s", value)
```

Comment:
In my log I'm getting the message twice (as warning and as error - I guess from the raised BadRequst):
`2022-02-17 13:29:14,894] DEBUG    - app.routes                 : GET /rest/services/ech/SearchServer
[2022-02-17 13:29:14,895] DEBUG    - flask_caching.backends.simplecache : get key 'get_topics_from_db' -> hit 
[2022-02-17 13:29:14,896] WARNING  - app.helpers.validation_search : Unsupported spatial reference 200
[2022-02-17 13:29:14,896] ERROR    - app                        : 400 Bad Request: Unsupported spatial reference 200
[2022-02-17 13:29:14,897] INFO     - app.routes                 : GET /rest/services/ech/SearchServer - 400 BAD REQUEST
[2022-02-17 13:29:14,898] INFO     - werkzeug                   : 127.0.0.1 - - [17/Feb/2022 13:29:14] "GET /rest/services/ech/SearchServer?sr=200&searchText=&lang=xx&type=locations&limit=2 HTTP/1.1" 400 -
`

Reply:
I guess it will be the same for the others with BadRequest